crates/typed-store/src/test_db.rs

```diff
@@ -1,841 +0,0 @@
-// Copyright (c) Mysten Labs, Inc.
```

Comment:
Sorry that I didn't state it before, but I didn't see that this file get's removed. IMO we should keep it, and only remove the keys and value and iterator parts that might not work anymore because they got removed by the 3 PRs. 
The iterator parts maybe can be replaced the same way it is done in the rest of the code.